change/@fluentui-react-a941c131-ba0e-40bc-a4b1-a87cd8554cff.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28907&commitId=d88f81b8217b7c8de9725e3afcc5fa23d0a6c1f4&env=prod&clientType=FLUENTUI)


  <details><summary><b>Image</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Image.Fit- cover, image taller.default.chromium.png|7603| Changed|
|Image.Fit- contain, image taller.default.chromium.png|1186| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>